src/delphi/eval/token_labelling.py

```diff
@@ -0,0 +1,184 @@
+# python -m spacy download en_core_web_sm
+# Should be run once, initially. Download only starts if not already installed.
+spacy.cli.download("en_core_web_sm", False, False, "-q")
+
+
+CATEGORIES = {
+    # custom categories
+    "Starts with space": (lambda token: token.text.startswith(" ")),  # bool
+    "Capitalized": (lambda token: token.text[0].isupper()),  # bool
+    # POS (part-of-speech) categories
+    # "POS Tag": (lambda token: token.pos_),  # 'NOUN', 'VB', ..
+    "Is Noun": (lambda token: token.pos_ == "NOUN"),  # redundant
+    "Is Pronoun": (lambda token: token.pos_ == "PRON"),  # redundant
+    "Is Adjective": (lambda token: token.pos_ == "ADJ"),  # redundant
+    "Is Verb": (lambda token: "VB" in token.tag_),  # redundant
+    "Is Adverb": (lambda token: token.pos_ == "ADV"),  # redundant
+    "Is Preposition": (lambda token: token.pos_ == "ADP"),  # redundant
+    "Is Conjunction": (lambda token: token.pos_ == "CONJ"),  # redundant
+    "Is Interjunction": (lambda token: token.pos_ == "INTJ"),  # redundant
+    # dependency categories
+    # "Dependency": (lambda token: token.dep_),  # 'nsubj', 'ROOT', 'dobj', ..
+    "Is Subject": (lambda token: token.dep_ == "nsubj"),
+    "Is Object": (lambda token: token.dep_ == "dobj"),
+    "Is Root": (
+        lambda token: token.dep_ == "ROOT"
+    ),  # root of the sentence (often a verb)
+    "Is auxiliary": (lambda token: token.dep_ == "aux"),  # redundant
+    # Named entity recognition (NER) categories
+    # "Named Entity Type": (lambda token: token.ent_type_),  # '', 'PERSON', 'ORG', 'GPE', ..
+    "Is Named Entity": (lambda token: token.ent_type_ != ""),
+}
+
+
+def explain_token_labels(token: Optional[Token] = None) -> None:
+    """
+    Prints the explanation of a specific token's labels or of ALL
+    possible labels (POS, dependency, NER, ...), if no token is provided.
+
+    Parameters
+    ----------
+    token : Optional[Token], optional
+        The token, whose labels should be explained. If None, all labels
+        possible labels are explained, by default None.
+    """
+    if token is not None:
+        # get token labels
+        labels = label_single_token(token)
+        print(" Explanation of token labels ".center(45, "-"))
+        print("Token text:".ljust(20), token.text)
+        print("Token dependency:".ljust(20), spacy.glossary.explain(token.dep_))
+        print("Token POS:".ljust(20), spacy.glossary.explain(token.pos_))
+        print(" Token labels ".center(45, "-"))
+        for i, (label, value) in enumerate(zip(CATEGORIES.keys(), labels)):
+            print(f" {i:2}  ", label.ljust(20), value)
+
+    else:
+        glossary = spacy.glossary.GLOSSARY
+        print(
+            f"Explanation of all {len(glossary.keys())} token labels (POS, dependency, NER, ...):"
+        )
+        for label, key in glossary.items():
+            print("   ", label.ljust(10), key)
+
+
+def label_single_token(token: Token) -> List:
```

Comment:
I would like to have function that just takes a `str` token label. Should we return this as a dict?